Python/compile.c

```diff
@@ -7186,14 +7186,16 @@ merge_const_one(struct compiler *c, PyObject **obj)
 }
 
 // This is in codeobject.c.
-extern void _Py_set_localsplus_info(int, PyObject *, _PyLocalsPlusKind,
-                                   PyObject *, _PyLocalsPlusKinds);
+extern void _Py_set_localsplus_info(int, PyObject *, unsigned char,
+                                   PyObject *, PyObject *);
 
-static void
-compute_localsplus_info(struct compiler *c, int nlocalsplus,
-                        PyObject *names, _PyLocalsPlusKinds kinds)
+static PyObject *
+compute_localsplus_info(struct compiler *c, PyObject *names)
```

Comment:
It seems strange to pass one in and return the other.  Perhaps create both in this function?

```suggestion
static int
compute_localsplus_info(struct compiler *c, int nlocalsplus, PyObject **pnames, PyObject **pkinds)
```

Reply:
...and something like "init_localsplus_info" is probably a better name with the shift in responsibility for this function.

Reply:
> It seems strange to pass one in and return the other. Perhaps create both in this function?

I find the C idiom for returning multiple values pretty awkward, so I'll just allocate both before passing them into the function.